src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/GroovyCallSiteSelectorTest.java

```diff
@@ -59,7 +59,7 @@
     }
 
     @Test public void overloads() throws Exception {
-        PrintWriter receiver = new PrintWriter(new NullOutputStream());
+        PrintWriter receiver = new PrintWriter(ByteStreams.nullOutputStream());
```

Comment:
Related somehow?

Reply:
Latest Guava doesn't support NullOutputStream. Deprecated for years.  Caused a failure on my machine - thought that might have been related to other changes that were removed.

Reply:
Jenkins runs Guava 11. Do not try to change that.

Reply:
@jglick 
Fixed.